zadania_ruby/fib.rb

```diff
@@ -0,0 +1,19 @@
+n = gets.to_i
```

Comment:
To nie działa, przypadkiem nie było do usunięcia?

Reply:
Jak nie działa, jak działa :) trzeba wpisać liczbę ile liczb ciągu ma się wyświetlić.